xbmc/Application.cpp

```diff
@@ -5010,12 +5010,36 @@ void CApplication::StopMusicScan()
     m_musicInfoScanner->Stop();
 }
 
-void CApplication::StartVideoCleanup(bool userInitiated /* = true */)
+void CApplication::StartVideoCleanup(bool userInitiated /* = true */,
+                                     const std::string& content /* = "" */)
 {
   if (userInitiated && CVideoLibraryQueue::GetInstance().IsRunning())
     return;
 
   std::set<int> paths;
+  if (!content.empty())
+  {
+    CVideoDatabase db;
+    std::set<std::string> content_paths;
+    if (db.Open() && db.GetPaths(content_paths))
+    {
+      for (const std::string& path : content_paths)
+      {
+        if (db.GetContentForPath(path) == content)
+        {
+          paths.insert(db.GetPathId(path));
+          std::vector<std::pair<int,std::string>> sub;
```

Comment:
space between type1 and type2 -> `std::pair<int, std::string>`

Reply:
done.